.shared_rubocop.yml

```diff
@@ -21,6 +21,7 @@ AllCops:
     - 'config/initializers/new_framework_defaults*'
     - 'spec/spec_helper.rb'
     - 'config.ru'
+    - 'bin/*'
```

Comment:
these are generated in the other projects but the rubocop rules will still run in rad_common because the AllCops is overridden in .rubocop.yml in the rad_common project